rep-2010.rst

```diff
@@ -220,7 +219,34 @@ And the definitions for ``example_interfaces/srv/AddTwoInts_Reponse`` is,
 Configuration
 -------------
 
-TODO: Describe what configuration options there are and how we plan to expose them (e.g. with ROS parameters and/or environment variables).
+Configuration of service introspection features will be done through ROS parameters.
+
+.. code-block:: yaml
+
+   /NODE_NAME:
+      ros__parameters:
+         
+         # Indicates if CLIENT_SENT and CLIENT_RECEIVED events will be published
+         SERVICE_NAME/client_events_enabled: bool
+
+         # Indicates if SERVICE_SENT and SERVICE_RECEIVED events will be published
+         SERVICE_NAME/service_events_enabled: bool
+         
+         # Indicates if request content will be included in CLIENT_SENT RequestEvent messages
+         SERVICE_NAME/_enable_client_sent_content: bool
+
+         # Indicates if request content will be included in SERVICE_RECEIVED RequestEvent messages
+         SERVICE_NAME/_enable_service_receive_content: bool
+
+         # Indicates if response content will be included in SERVICE_SENT RequestEvent messages
+         SERVICE_NAME/_enable_service_sent_content: bool
+
+         # Indicates if response content will be included in CLIENT_RECEIVED RequestEvent messages
+         SERVICE_NAME/_enable_client_receive_content: bool
+         
+         # Indicates if service introspection will be enabled for default services
+         _enable_default_service_introspection: bool 
```

Comment:
I think this parameter becomes obsolete if we decide to omit the service name prefix from parameters.